components/analytics-processors/org.wso2.carbon.analytics.spark.ui/src/main/java/org/wso2/carbon/analytics/spark/ui/client/AnalyticsExecutionClient.java

```diff
@@ -96,9 +96,12 @@ private String errorToJson(String query, String message) {
         return resObj.toString();
     }
 
-    public AnalyticsProcessorAdminServiceStub.AnalyticsQueryResultDto[] executeScriptContent(String scriptContent) throws RemoteException,
-            AnalyticsProcessorAdminServiceAnalyticsProcessorAdminExceptionException {
-        return stub.execute(scriptContent);
+    public AnalyticsProcessorAdminServiceStub.AnalyticsQueryResultDto[] executeScriptContent(String scriptContent) throws Exception {
```

Comment:
It seems like that this line length exceeds the standard length. please reformat it.

Reply:
Code was updated in commit [#76ac8654774a8592444c803dcb7ba80d640e2d2e](https://github.com/wso2/carbon-analytics/commit/76ac8654774a8592444c803dcb7ba80d640e2d2e)